CONTRIBUTING.md

```diff
@@ -31,10 +31,10 @@ Below are some loose requirements we'd like all PR's to follow. Our release proc
 
 ### Quality assurance and code reviews
 
-All PRs will be tested, vetted, and reviewed by our team before being merged with the main code base. All should be pull requested into whatever the upcoming release branch is. Find that by searching for the highest SEMVER `release-X.Y.Z` branch or following our release documentation.
+All PRs will be tested, vetted, and reviewed by our team before being merged with the main code base. All should be pull requested into the `develop` branch. 
 
-### Steps to submit a PR
-	- All PRs should request merges back into LME's *CLOSEST* Major or Minor upcoming release branch `release-X.Y.Z`. This will be viewable in the branch list on Github. You can also refer to our release documentation for guidance. 
+### Steps to submit a PR  
+  - All PRs should request merges back into LME's `develop` branch.  This will be viewable in the branch list on Github. You can also refer to our release documentation for guidance. If the fix fits the requirements for a hotfix, the LME team will modify your PR as is relevant.  
```

Comment:
```suggestion
  - All PRs should request merges back into LME's `develop` branch.  This will be viewable in the branch list on Github. You can also refer to our [release documentation](https://github.com/cisagov/LME/blob/main/RELEASES.md) for guidance. If the fix fits the requirements for a hotfix, the LME team will modify your PR as is relevant.  
```